src/deluge/gui/ui/menus.cpp

```diff
@@ -1508,7 +1508,23 @@ MenuItem* paramShortcutsForSounds[][8] = {
     {&lfo2RateMenu,           &lfo2SyncMenu,           &lfo2TypeMenu,                  &reverbAmountMenu,              &reverbPanMenu,       &reverbWidthMenu,       &reverbDampingMenu,       &reverbRoomSizeMenu                },
     {&delayRateMenu,          &delaySyncMenu,          &delayAnalogMenu,               &delayFeedbackMenu,             &delayPingPongMenu,   nullptr,                nullptr,                  nullptr                            },
 };
-
+Submenu* parentsForSoundShortcuts[][8] = {
```

Comment:
just a nit, I know you were copying the other arrays, but can we replace "8" with kDisplayHeight?